Company-Project/frontend/src/hypernova.js

```diff
@@ -0,0 +1,98 @@
+const hypernova = require('hypernova/server');
+const renderReact = require('hypernova-react').renderReact;
+const Sentry = require('@sentry/node');
+// const path = require("path");
+
+require = require("esm")(module/*, options*/)
+
+const moduleAlias = require('module-alias')
+moduleAlias.addAliases({
+  'utils': __dirname + '/utils',
+  'styles': __dirname + '/styles',
+  'components': __dirname + '/components',
+})
```

Comment:
In my opinion we should get rid of this before merging like we talked about. IDE discoverability > import looks :)

Reply:
Good point! Also the configuration is taken place in two files, this and the vscode editor config, so removing alias imports simplifies the implementation. Win win.